xapian-applications/omega/templates/query

```diff
@@ -153,3 +153,4 @@ $map{$cgilist{B},<input type=hidden name=B value="$html{$_}">}
 </body>
 </html>
 $log{query.log}
+$log{search.log,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}	$csv{$query,1}	$hitlist{$id$.}	$topdoc}
```

Comment:
With TSV it seems we have to either mangle query strings to eliminate tabs (e.g. changing tabs to spaces) or else define our own variant of TSV with special rules for quoting values containing tabs.  Not being able to preserve queries exactly as typed by the user seems inelegant, while defining our own variant rather ruins the interoperability a standard format ought to give us (because you can't expect existing libraries to handle quoted tabs).

CSV seems a better option as it can at least handle arbitrary query strings without having to mangle them, and omegascript supports CSV escaping already.